core/src/main/java/io/servicecomb/core/Invocation.java

```diff
@@ -74,6 +74,9 @@ private void init(OperationMeta operationMeta, Object[] swaggerArguments) {
   }
 
   public Transport getTransport() {
+    if (endpoint == null) {
+      throw new IllegalStateException("Endpoint is empty. Forget to configure \"loadbalance\" in consumer handler chain?");
```

Comment:
Can we change the message to "Endpoint is empty. Please  configure \"loadbalance\" in the consumer handler chain.

Reply:
I guess this exception should be detected at startup time and should not be thrown at run time. Validate the `microservice.yaml` file before the application starts.

Reply:
we don't know which handler is loadBalance
because cutomer can dev their own loadBalance

Reply:
Yes, we don't know at which handler will a developer to set the address, and they can not use 'loadbalance' and define a address resolver themself. So this message is a best guess for the developer.